core/src/main/dotnet/src/org/apache/spark/ml/feature/StringIndexerModel.cs

```diff
@@ -0,0 +1,227 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+
+using System;
```

Comment:
These are generated but manually changed some like the constructors. These are needed because some of our tests depend on them and the official Microsoft.Spark hasn't incorporated them. We might want to auto-generate most of ML classes for Microsoft.Spark and once they publish them we can remove these from our repo and depend on Microsoft.Spark.